cyral/resource_cyral_sidecar_listener.go

```diff
@@ -402,5 +418,22 @@ func getSidecarListenerSchema() map[string]*schema.Schema {
 				},
 			},
 		},
+		OverrideRepoClientTlsSettingsKey: {
+			Description: "Override TLS settings defined in the repo. " +
+				"Default value generated by API if not provided. ",
+			Type:     schema.TypeBool,
+			Optional: true,
+			Computed: true,
+		},
+		TlsModeKey: {
+			Description: "TLS mode. " +
+				"Default value generated by API if not provided. " +
+				"Note! This field is in effect only if OverrideRepoClientTlsSettings is set to true or the listener " +
+				"is a SMART port (enabled in more than one binding). " +
```

Comment:
**Suggestion:** Is there a link in our public doc regarding smart ports? Might be interesting to add this link in this description as a reference for customers that want to understand how it works. I would also recommend adding a reference to the `cyral_repository_binding` resource here, something like:
```suggestion
				"is a SMART port (enabled in more than one binding - see [`cyral_repository_binding`](../resources/repository_binding.md)). " +
```

Reply:
Regarding the link to our public docs, please refer to the latest version instead of some specific release.